src/modules/registrypreview/RegistryPreviewUI/MainWindow.Utilities.cs

```diff
@@ -1028,8 +1035,11 @@ private async void SaveWindowPlacementFile(string path, string filename)
 
             try
             {
-                fileContents = jsonWindowPlacement.Stringify();
-                await Windows.Storage.FileIO.WriteTextAsync(storageFile, fileContents);
+                if (jsonWindowPlacement != null)
```

Comment:
Nothing more than throw protection on the first run or if the file goes missing.  No impact to the end user either way but it does help when you're trying to debug something on a first time install.